test/functional/http/post/3.votes.js

```diff
@@ -483,6 +484,10 @@ describe('POST /api/transactions (type 3) votes', () => {
 		phases.confirmation(goodTransactions, badTransactions);
 	});
 
+	// TODO: These test cases broke after removing support for batch HTTP POST
+	// transactions and only allowing a single POST transaction per request.
+	// The issue doesn't appear to be related. Maybe a race condition?
+	// See https://github.com/LiskHQ/lisk/issues/1427
```

Comment:
If these tests are really failing in this PR, then we should fix these right away.  @diego-G can help out to understand failure for these. If there is quick fix, we should do in this PR.

Reply:
It's just a few difficult cases. It's hard to know what's causing the issues because each test case in this file builds on top of the previous one. We could look at this as part of this issue or raise a different issue about this since a relatively small number of cases are affected.